src/applications/coronavirus-vaccination/api/index.js

```diff
@@ -9,21 +10,25 @@ const unsubscribeUrl = `${
 export const retrievePreviouslySubmittedForm = () => apiRequest(apiUrl);
 
 export const saveForm = formData => {
+  const csrfTokenStored = localStorage.getItem('csrfToken');
```

Comment:
Would it make sense to put this variable up with apiUrl and unsubscribeUrl since it is used in both functions?

Reply:
Moving the variable declaration makes sense to me as well, unless there is potentially some cacheing or other issue where we need to ensure a new `csrfToken` is pulled every time 🤷 , I am not sure!

Reply:
I _think_ we want to look it up from local storage right before API request since other requests may have updated the csrfToken.

Reply:
If that's the case, it makes sense to have it inside the functions, then. Just wanted to check. 😄